README.md

```diff
@@ -4,10 +4,14 @@
 
 The `nextstrain.yml` file in this repo is hosted at <http://data.nextstrain.org/nextstrain.yml> and serves as an environment definition for Conda.
 
-You can set it up by running:
+We strongly recommend [Mamba](https://github.com/mamba-org/mamba) as a faster, drop-in replacement for the default Conda installer.
+
+    conda install -c conda-forge mamba
+
+After installing Mamba, setup the Nextstrain environment by running:
 
     curl http://data.nextstrain.org/nextstrain.yml -o nextstrain.yml
-    conda env create -f nextstrain.yml
+    mamba env create -f nextstrain.yml
     conda activate nextstrain
```

Comment:
I'm not sure about this change, as it feels to me like it introduces another place for confusion in the process. For example, first you have to use Conda to install Mamba. Then you use Mamba to install Nextstrain. But _then_ you go back to Conda again to activate Nextstrain. I understand the nuances here, but introducing Mamba seems it might add confusion for many folks.

How much faster is Mamba for this install? For the typical person, they might only create the environment once or twice. Is it worth it?

(and anyone who already understands the nuances because they use Mamba already, will know that they can use `mamba env create` instead of `conda env create` even if the instructions don't say so.)

Reply:
I see what you're saying. It's hard to benchmark the time savings without deleting miniconda and starting from scratch under both scenarios and even then I'm not sure I can consistently recreate the issues I've had in the past with conda-forge. I think the bind we're in is:

1. Bioconda expects conda-forge to be an available channel for all non-biology-related dependencies (and Snakemake relies on this extra channel, too)
2. Adding conda-forge as a channel can be installations either painfully slow or [completely broken](https://github.com/conda/conda/issues/9905) or [both](https://github.com/nextstrain/augur/pull/411)

[We removed conda-forge from this environment](https://github.com/nextstrain/conda/commit/0ad32bc19a6a3e91d812bfbfd1aeeee7f63f05ec) for the last reason.

In practice, Mamba has solved this issue as an independent tool, but those solutions haven't made their way back into Conda and it's not clear if they ever will. This is why [Snakemake recommends Mamba for its own installation](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html#installation-via-conda); it's frustrating to have to work around the package manager itself, but the benefit of being able to consistently install your packages is worth the added complexity of yet another executable.

In terms of minimizing conceptual dependencies for users who get to this README, I don't see `mamba` as more complicated than `curl` or `conda`. My guess is that folks are either going to copy and paste (hoping for the best) or, if they know the nuances, they will copy and paste knowing that it is the better solution for them.

Maybe as an alternative to my current proposal, we could reword the instructions to align with [the current Augur installation docs](https://docs.nextstrain.org/projects/augur/en/stable/installation/installation.html#using-conda)? These suggest Mamba as an alternative for people who want to try it instead of making it the required path.

Reply:
db07cda revises the README to make Mamba an alternate path to installation while retaining the current full-Conda approach.

Reply:
Re: conda-forge, see the discussion in bioconda/bioconda-utils#330

Reply:
Thank you, @fanninpm! That is very helpful to know that our original channel order may have caused our slow/broken installations in the past.

Reply:
@huddlej +1 for keeping vanilla Conda for now and including parallel instructions with Mamba.